SimCalorimetry/HGCalSimProducers/src/HGCHEbackDigitizer.cc

```diff
@@ -22,15 +22,19 @@ namespace {
   void addCellMetadata(HGCCellInfo& info,
```

Comment:
this entire function seems identical to the one in `HGCDigitizerBase.cc` - why can't that one be reused?

Reply:
This we try to resolve later